content/docs/setup/kubernetes/upgrade/steps/index.md

```diff
@@ -221,12 +223,15 @@ spec:
 trafficPolicy:
   tls:
     mode: DISABLE
+---
 {{< /text >}}
 
 ## Migrating the `mtls_excluded_services` configuration to destination rules
```

Comment:
Same comment as above, is this relevant for 1.1 -> 1.2?

Reply:
I'm pretty sure all these security related stuff should be removed.